### ISSUE TITLE
Graphs have insufficient text alternatives [Android]

### ISSUE THREAD

**Issue by jonnyjames** [covid-alert-ny/covid-green-app#232]
The graphs found on the “NY COVID Data” screen have insufficient text alternatives. For example, the “Total persons tested” graph has an accessible name of “76997 undefined, Graph showing daily tests over time”. The actual data of the graph is unavailable to TalkBack users. For example, the accessible name could be “Daily tests over time, August 2nd [number of persons tested], August 8th [number of persons tested], August 14th [number of persons tested], August 20th [number of persons tested], August 26th [number of persons tested], August 30 [number of persons tested]”. The example accessible name doesn’t convey all the graph data (i.e., only data for 6 days out of the month), but providing data for each day would make the accessible name excessively verbose.

![data Tab - Image - Chart alt](https://user-images.githubusercontent.com/7034772/92009722-ed07db80-ed16-11ea-918f-9a8b0baa527b.png)

Severity: Medium

**Comment by AlanSl** [comment]
Already reported in https://github.com/project-vagabond/covid-green-app/issues/198

https://github.com/project-vagabond/covid-green-app/pull/205 fixes it but is awaiting review since yesterday

**Comment by AlanSl** [comment]
Merged, should be fixed in next build

**Comment by jonnyjames** [comment]
@floridemai This issue is now fixed. I am closing this issue.
